osh/hub/osh-hub-httpd.conf

```diff
@@ -18,23 +18,20 @@ WSGISocketPrefix /var/run/wsgi
     SSLCertificateKeyFile   /etc/httpd/conf/localhost.key
 
     # WSGI handler
-    WSGIDaemonProcess covscan display-name=%{GROUP} locale='C.UTF-8'
-    WSGIProcessGroup covscan
-    WSGIScriptAlias /osh /usr/lib/python3.6/site-packages/osh/hub/osh-hub.wsgi process-group=covscan
+    WSGIDaemonProcess osh display-name=%{GROUP} locale='C.UTF-8'
+    WSGIProcessGroup osh
+    WSGIScriptAlias /osh @PYTHON3_SITELIB@/osh/hub/osh-hub.wsgi process-group=osh
```

Comment:
Should we rename the file to `osh-hub-httpd.conf.in` to make it clear that it is a template and not something that will work out of the box?

Alternatively, we could keep `/usr/lib/python3.6` as it is and substitute this string in the spec file.

Reply:
This still needs to be addressed.

Reply:
Hmm, I'm not entirely sure if a file can be easily renamed using `setup.py` (to drop the `.in` suffix).  @kdudka,  would it be ok if the file was installed manually in the `%install` section instead?

Reply:
Yes, we can drop the change of `setup.py` and take the template file directly from the build directory in `%install`.

Reply:
Addressed in 3e572db and f33ed48.